src/main/java/com/barszcz/server/controller/RoomController.java

```diff
@@ -0,0 +1,83 @@
+package com.barszcz.server.controller;
+
+import com.barszcz.server.dao.RoomsDao;
+import com.barszcz.server.entity.Respond;
+import com.barszcz.server.entity.RoomModel;
+import com.barszcz.server.entity.RoomRespondModel;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class RoomController {
+
+    private final RoomsDao roomsDao;
+    private final SimpMessagingTemplate simpMessagingTemplate;
+
+    @MessageMapping("/addRoom")
+    @SendTo("/rooms/change")
+    public List<RoomModel> broadcastRooms(String message) throws Exception {
+        ObjectMapper objectMapper = new ObjectMapper();
+        RoomRespondModel roomRespondModel = objectMapper.readValue(message, RoomRespondModel.class);
+        if (roomsDao.findRoomModelByRoomLike(roomRespondModel.getRoom()).isEmpty()) {
```

Comment:
Takiej logiki raczej trzeba uniknąć w controlerze. Stwórz serwis który bedzie za to odpowiadał

Reply:
I zrób to na litość boską w Optionalach

Reply:
isEmpty aż się prosi o NPE